telegram/bot.py

```diff
@@ -1170,20 +1183,21 @@ def edit_message_live_location(self,
                 specified. Unique identifier for the target chat or username of the target channel
                 (in the format @channelusername).
             message_id (:obj:`int`, optional): Required if inline_message_id is not specified.
-                Identifier of the sent message.
+                Identifier of the message to edit.
             inline_message_id (:obj:`str`, optional): Required if chat_id and message_id are not
                 specified. Identifier of the inline message.
             latitude (:obj:`float`, optional): Latitude of location.
             longitude (:obj:`float`, optional): Longitude of location.
             location (:class:`telegram.Location`, optional): The location to send.
             reply_markup (:class:`telegram.InlineKeyboardMarkup`, optional): A JSON-serialized
-                object for an inline keyboard.
+                object for a new inline keyboard.
             timeout (:obj:`int` | :obj:`float`, optional): If this value is specified, use it as
                 the read timeout from the server (instead of the one specified during creation of
                 the connection pool).
 
         Returns:
              :class:`telegram.Message`: On success the edited message.
+             :obj:`bool`: ``True`` is returned on failure.
```

Comment:
`:obj:'True'` is returned, when the message to be edited was not sent by the bot. True wont be returned on failure ;)

Reply:
Oh right I'll change that